docker/Dockerfile

```diff
@@ -14,10 +14,8 @@ COPY zu /zu
 WORKDIR /zu
 RUN ./gradlew assemble
 
-FROM zookeeper:3.5.5
+FROM zookeeper:3.5.4-beta
```

Comment:
Is there a reason to go back to `3.5.4-beta`?

Reply:
@baluchicken I think https://github.com/apache/zookeeper/pull/1254 may address this issue

Reply:
Hi guy! I created the PR referenced by @amuraru . The issue we are fixing in ZooKeeper with this PR should be present both in 3.5.5 and 3.5.4-beta. So maybe there is an other reason why you want to downgrade to this specific beta version...